src/Components/ObjectProperties/ObjectProperties.js

```diff
@@ -46,12 +46,18 @@ class ObjectProperties extends React.Component {
     super(props);
 
     this.state = {
-      keyWidth: null,
+      keyWidth: this.props.keyWidth || null,
     };
 
     this.keyWidthsIndex = {};
   };
 
+  componentWillReceiveProps = (nextProps) => {
+    if (nextProps.keyWidth) {
+      this.setState({keyWidth: nextProps.keyWidth});
```

Comment:
@XVincentX 

But this is called during the lifecycle of the Attributes Kit, not with the initial rendering of AK.